eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -78,12 +78,24 @@ function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata
     $ReadmeContent = $ReadmeContent -replace $releaseReplaceRegex, $replacementPattern
   }
   
+  # Get the first code owners of the package.
+  $author = "ramya-rao-a"
+  $msauthor = "ramyar"
+  #$codeOwnerFilePath = Join-Path (Get-Item $PSScriptRoot ).Parent.Parent.Parent.FullName ".github/CODEOWNERS"
+  $codeOwnerArray = ."$PSScriptRoot/get-codeowners.ps1" `
+                      -TargetDirectory $PackageInfo.DirectoryPath 
+                      #-CodeOwnerFileLocation $codeOwnerFilePath
+  if ($codeOwnerArray) {
```

Comment:
https://github.com/Azure/azure-sdk-tools/blob/main/tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners/Program.cs#L26-L30

Looks like we need to update the tool if we want no matches found to return empty list. Currently it will fail.